### ISSUE TITLE
Programming Blog Link Broken - [Solved with Prefetch]

### ISSUE THREAD

**Issue by apophis51** [apophis51/next-js-Portfolio#49]
We have a new issue where clicking 'PROGRAMMING BLOGS' on deployed version leads to the error " Application error: a client-side exception has occurred (see the browser console for more information)."

![image](https://github.com/apophis51/next-js-Portfolio/assets/49728948/89f26210-c974-4d0d-adab-6816d1fe83e7)

code in Navigation > page.js currently reads:

`   <MenuItem key={page.name} onClick={handleCloseNavMenu}>
                  {page.href == '/Crypto/GuessTheNumber' ? (<Link href="/Crypto/GuessTheNumber" prefetch={false}>
                  <Typography textAlign="center"> {page.name}</Typography>
    </Link>) : (<Link href={page.href} ><Typography textAlign="center">{page.name}</Typography></Link>)}
                </MenuItem>`

console.log on the browser is repporting
`290-09bdaf52166e1cb8.js:1 ChunkLoadError: Loading chunk 738 failed.
(error: https://profile.malcmind.com/_next/static/chunks/app/(main%2520site)/BlogCards/page-399eb59676fd545b.js?dpl=dpl_4b8JfVSmJfrbMECu3eRtFZNJfaZp)
    at __webpack_require__.f.j (webpack-a2bac6df6f6c472e.js:1:4093)
    at webpack-a2bac6df6f6c472e.js:1:1761
    at Array.reduce (<anonymous>)
    at __webpack_require__.e (webpack-a2bac6df6f6c472e.js:1:1709)
    at 290-09bdaf52166e1cb8.js:17:7023
    at 290-09bdaf52166e1cb8.js:17:7225
    at 290-09bdaf52166e1cb8.js:17:7428
    at c (290-09bdaf52166e1cb8.js:17:8974)`

however a direct load of https://profile.malcmind.com/BlogCards causes no issue

![image](https://github.com/apophis51/next-js-Portfolio/assets/49728948/46dbea42-e595-426e-9023-70fb3b0c66e1)

Not quite sure why this is now an issue but suggest solution will be setting prefetch to false to force the render as follows: 

`<MenuItem key={page.name} onClick={handleCloseNavMenu}>
                  {page.href == '/Crypto/GuessTheNumber' ? (<Link href="/Crypto/GuessTheNumber" prefetch={false}>
                  <Typography textAlign="center"> {page.name}</Typography>
    </Link>) : (<Link href={page.href} prefetch={false}><Typography textAlign="center">{page.name}</Typography></Link>)}
                </MenuItem>`

**Comment by apophis51** [comment]
I conquer with my own reasoning. Lets go ahead and implement the code edit to see if the issue resolves.

**Comment by apophis51** [comment]
having difficulty compling the program with the new edits we keep gettting A Serverless Function has exceeded the unzipped maximum size of 250 MB.  going to . ignore the scripts file to see if that helps

**Comment by apophis51** [comment]
Ignoring the scripts file had no effect. trying to set prefetch back to true to see if that remedies the failure to compile

**Comment by apophis51** [comment]
serveless function error is still persisting after the revert

**Comment by apophis51** [comment]
solution might be to pour gasoline on the program and let it burn... will continue debugging tomrorow

**Comment by apophis51** [comment]
To solve the "A Serverless Function has exceeded the unzipped maximum size of 250 MB" I tried deleting aws-sdk, and wagmi because they took more then 100MB of data on node modules when I did "du -ah --max-depth=10 nextjs-flask | sort -h" however that did not fix the issue. When I query the .next build file its over 300MB.  I realized that I accidently left api visible and quickly changed it to .api   That fixed the Serverless function issue.   My local .next still reads over 300MB but when deploying it says build cache is 233MB


The initial problem was fixed by changing these two code snippets to use prefetch:

` {page.href == '/Crypto/GuessTheNumber' ? (<Link href="/Crypto/GuessTheNumber" prefetch={false}>
      {page.name}
    </Link>) : (<Link href={page.href} prefetch={false}>{page.name}</Link>)} `
`  <MenuItem key={page.name} onClick={handleCloseNavMenu}>
                  {page.href == '/Crypto/GuessTheNumber' ? (<Link href="/Crypto/GuessTheNumber" prefetch={false}>
                  <Typography textAlign="center"> {page.name}</Typography>
    </Link>) : (<Link href={page.href} prefetch={false}><Typography textAlign="center">{page.name}</Typography></Link>)}
                </MenuItem>`
